singular.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The template for displaying all pages, single posts and attachments
+ *
+ * This is a new template file that WordPress introduced in
+ * version 4.2. Note that it uses conditional logic to display
```

Comment:
This template was introduced in 4.3, wasn't it? Although it won't give us a fatal error, we probably should stop the theme to be activated below 4.3 to make sure the theme work as it supposed to be.